src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Views/Controls/SampleControlInfoAttribute.cs

```diff
@@ -6,36 +6,60 @@ namespace Uno.UI.Samples.Controls
 {
 
 	[AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = true)]
```

Comment:
`AllowMultiple = true` ?

Reply:
Agree it's a non sense, that's why I set it to false on the new attribute. But I won't change the old one.